src/js/directives/uiselect.js

```diff
@@ -14,15 +14,27 @@ angular.module('xeditable').directive('editableUiSelect',['editableDirectiveFact
             return newEl;
         };
 
-        var match = null;
-        var choices = null;
+        var findElement = function(name) {
```

Comment:
I would better make that function return an `index` rather than changing variable.
for example `var index = findElement(this.name)`